src/Datadog.Trace/Logging/DatadogLogging.cs

```diff
@@ -94,6 +94,18 @@ public static ILogger For<T>()
             return GetLogger(typeof(T));
         }
 
+        public static void SafeLogError(this ILogger logger, Exception ex, string message, params string[] args)
+        {
+            try
+            {
+                logger.Error(ex, message, args);
+            }
+            catch
+            {
+                // ignore
```

Comment:
Should we at least output to stdout? I have reservations about completely ignoring exceptions.

Reply:
Sure, IMO, it's a best effort reporting. 
Number one priority is not to crash apps.